Build/Build.csproj

```diff
@@ -48,14 +48,14 @@
     <None Include="Remotion.build" />
     <None Include="Remotion.Local.build" />
     <None Include="Remotion.Server.build" />
-    <None Include="TestingSetupForDatabase.build" />
+    <None Include="TestSetupForDatabase.build" />
   </ItemGroup>
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Remotion.BuildScript, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Remotion.BuildScript.2.0.0-alpha018\lib\net45\Remotion.BuildScript.dll</HintPath>
+    <Reference Include="JetBrains.Annotations, Version=2019.1.3.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325">
```

Comment:
That's ood. Why is the BuildScript no longer referenced?

Reply:
Probably because the Build project is excluded from build in the solution and Rider therefore removed the reference....?

Reply:
*blinks*

Oookay...please fix